schemas/field.json

```
@@ -193,7 +193,17 @@
                         },
                         "value": {
```

```suggestion
                        "values": {
```

Following https://github.com/ideditor/schema-builder/pull/87#issuecomment-2131163146 lets make this a breaking change.